From 88f9b7ab5e2f99054b8f3a4173296e0cb9129067 Mon Sep 17 00:00:00 2001
From: Adriana Kobylak <anoo@linux.vnet.ibm.com>
Date: Sun, 30 Jul 2017 16:13:43 -0500
Subject: [PATCH u-boot v2016.07-aspeed-openbmc] Add MTD and UBI support to ast-g5

Add MTD and UBI support to the default g5 include, conditioned to
having MTDPARTS_DEFAULT defined.

This allows platforms to enable this support if desired without
adding it by default since it increases the size of u-boot.

Signed-off-by: Adriana Kobylak <anoo@linux.vnet.ibm.com>
---
 include/configs/ast-g5-ncsi.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/include/configs/ast-g5-ncsi.h b/include/configs/ast-g5-ncsi.h
index 12d6684..b126e6e 100644
--- a/include/configs/ast-g5-ncsi.h
+++ b/include/configs/ast-g5-ncsi.h
@@ -28,4 +28,14 @@
 
 #define CONFIG_HW_WATCHDOG
 
+/* MTD and UBI */
+#ifdef MTDPARTS_DEFAULT
+#define CONFIG_FLASH_CFI_MTD
+#define CONFIG_CMD_UBI
+#define CONFIG_RBTREE
+#define CONFIG_MTD_DEVICE
+#define CONFIG_MTD_PARTITIONS
+#define CONFIG_CMD_MTDPARTS
+#endif
+
 #endif	/* __AST_G5_NCSI_CONFIG_H */
-- 
1.8.2.2

